tests/unit/exceptions/test-class-wc-payments-amount-too-low-exception.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Class WC_Payments_Amount_Too_Low_Exception_Test
+ *
+ * @package WooCommerce\Payments\Tests
+ */
+
+use WCPay\Exceptions\Amount_Too_Low_Exception;
+
+/**
+ * Test cases for Amount_Too_Low_Exception
+ */
+class WC_Payments_Amount_Too_Low_Exception_Test extends WP_UnitTestCase {
+
+	public function test_getMessage() {
+		$e_without_data = new WCPay\Exceptions\Amount_Too_Low_Exception( 'Test', 'wcpay_amount_too_low', 400, [] );
+		$this->assertEquals(
+			__(
+				'The specified amount is less than the minimum amount allowed. Use a higher amount and try again.',
+				'woocommerce-payments'
+			),
+			$e_without_data->getMessage()
+		);
+
+		$error_data = [
+			'currency'   => 'usd',
+			'min_amount' => 60,
+		];
+
+		$e_with_data = new WCPay\Exceptions\Amount_Too_Low_Exception( 'Test', 'wcpay_amount_too_low', 400, $error_data );
+		$this->assertEquals(
+			sprintf(
+				// translators: %1$s is a formatted amount with currency code.
+				__(
+					'Sorry, the minimum allowed order total is %1$s to use this payment method.',
+					'woocommerce-payments'
+				),
+				wc_price( $error_data['min_amount'] / 100, [ 'currency' => strtoupper( $error_data['currency'] ) ] )
```

Comment:
Since this is known value in test, we might even use `$0.60` inside message without sprintf :)

Reply:
`wc_price` does some wrapping with `<span>` elements and it might differ from version to version, hence I use it in the tests to match generated message in the exception.

Reply:
That's even bigger question, do we need this span in error message and anywhere else?

Reply:
Yeah, an interesting question indeed. If we consider `API_Exception` and its derivatives as something that can be directly communicated to the user, then I see no issue with `<span>` as long as it is properly documented. Other use cases can strip HTML tags if needed.

Reply:
I'm still look at this addition of span as a side effect. I wasn't aware of it and didn't expect it until you've mentioned the span is added. Addressing this globally might be out of scope of this PR though. Anyway, I'd prefer us using literal strings for setting expectations rather than something calculated with the same utils we use in the code under test.

Reply:
Addressed in the related change to price formatting method.